backend/server.py

```diff
@@ -52,12 +52,17 @@ def get_all_students():
 @app.route('/student')
 def get_particular_student():
     username = request.args.get('username')
-    if username is None or username is '':
+    roll = request.args.get('roll')
+    if ((username == None or username == '' )and (roll == None or roll == '')) :
```

Comment:
- Use `is` instead of `==` to maintain consistency in the codebase
- Add space before `and`
- Remove space before `:`